docker-compose.yml

```diff
@@ -1,72 +1,96 @@
 version: '2'
 services:
   rabbit:
-     image: rabbitmq:3-management
-     env_file:
-       - rabbit.env
-     ports:
+    image: rabbitmq:3-management
```

Comment:
It's kind of difficult to see where the work has actually be done here given there's a lot of whitespace amends. Can we make sure we're not auto formatting files and keep PR content to only relevant work?